index.html

```diff
@@ -190,10 +190,10 @@ <h2>I'm always interested in hearing about new projects, so if you'd like to cha
         </div>
         
         <form action="#" class="footer-form">
-            <input class="input" type="text" placeholder="Full name">
-            <input class="input" type="email" placeholder="Email address">
-            <textarea name="" id="" cols="30" rows="10">Enter text here</textarea>
-            <button class="abt-btn" type="button">Get in touch</button>
+            <input class="input" type="text" placeholder="Full name" maxlength="30" required>
+            <input class="input" type="email" placeholder="Email address" required>
+            <textarea name="" id="" cols="30" rows="10" maxlength="500">Enter text here</textarea>
+            <button class="abt-btn" type="submit">Get in touch</button>
         </form>
```

Comment:
Great job @Tekhlay 🤝

- [ ] Kindly make sure the form uses the action and method attribute to integrate with [formspree service](https://formspree.io/html). `action` value should be similar to `https://formspree.io/f/{form_id}` and `method` value equals `POST`.
- [ ] Kindly make the `Enter text here` a placeholder for the `Textarea` field.
- [ ] You may also add the `required` attribute to the `Textarea` field.